node/src/commands/genesis/mod.rs

```diff
@@ -166,6 +166,8 @@ fn create_accounts(
             }
         }
 
+        account_data.account.set_nonce(Felt::new(1))?;
```

Comment:
minor nit: we can import `ONE` constant from `miden-base` and use it here instead of `Felt::new(1)` (that's how we usually do it).

Reply:
Questions:
- I saw that we had `ONE` and `ZERO`, curious why we have them. Are they simply shorthands to `Felt::new(0) and 1` or are they useful in another manner? Why only `ONE` and `ZERO`? 
- We can import as an example `ONE` but also many other types from `miden-objects` but also `miden-crypto` why is that? Should we find a way to make sure that all crates depend only on `miden-objects` for global types?

Reply:
> I saw that we had `ONE` and `ZERO`, curious why we have them. Are they simply shorthands to `Felt::new(0) and 1` or are they useful in another manner? Why only `ONE` and `ZERO`?

Yes - these are just shorthands. We defined only these ones as they are used quite frequently in various places.

> We can import as an example `ONE` but also many other types from `miden-objects` but also `miden-crypto` why is that? Should we find a way to make sure that all crates depend only on `miden-objects` for global types?

Yes - the intent is so that most crates only need to depend on `miden-base` and are not even aware of `miden-crypto`. Doesn't always work out this way, but I try to make it work whenever possible.